core/src/main/java/org/elasticsearch/index/query/MultiMatchQueryBuilder.java

```diff
@@ -179,7 +193,7 @@ public MultiMatchQueryBuilder field(String field, float boost) {
     /**
      * Sets the type of the text query.
      */
-    public MultiMatchQueryBuilder type(MultiMatchQueryBuilder.Type type) {
+    public MultiMatchQueryBuilder type(Type type) {
```

Comment:
I agree Type is very used name space, so MatchQuery.Type makes it more explicit.